accounts/models.py

```diff
@@ -153,6 +153,7 @@ class Account(models.Model):
 
     note = models.TextField()
     url = models.URLField(db_index=True)
+    activitypub_id = models.URLField(null=True)
```

Comment:
```suggestion
    activitypub_id = models.URLField(null=True, blank=True)
```

Just in case I have to mess around in Django Admin.